dandiapi/api/views/auth.py

```diff
@@ -96,5 +97,9 @@ def user_questionnaire_form_view(request: HttpRequest) -> HttpResponse:
     return render(
         request,
         'api/account/questionnaire_form.html',
-        {'questions': QUESTIONS, 'query_params': request.GET.dict()},
+        {
+            'questions': QUESTIONS,
+            'query_params': request.GET.dict(),
+            'dandi_web_app_url': settings.DANDI_WEB_APP_URL,
```

Comment:
Why is this included now?

Reply:
[The base template that this one extends uses it](https://github.com/dandi/dandi-api/blob/use-django-sites/dandiapi/api/templates/api/account/base.html)